packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -115,7 +119,7 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
     disabled: boolean,
```

Comment:
again...unless this is something new that I'm not familiar with, we aren't using getClassnames any longer either. `getStyles = {(props) => {root: 'myClassName'}}` works just fine to supply custom classnames.